<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>popupMovieDetails</class>
 <widget class="QDialog" name="popupMovieDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  border: 2px solid rgb(49, 45, 46)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Black Red Movie Apps Presentation Template (1).jpg</pixmap>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
 border: 2px solid rgb(49, 45, 46);
    border-radius: 20px;
}
</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
 border: 2px solid rgb(49, 45, 46);
    border-radius: 20px;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
 border: 2px solid rgb(49, 45, 46);
    border-radius: 20px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="pathDisplay">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>65</y>
     <width>161</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
